### DISCUSSION TITLE
OBS Help

### DISCUSSION THREAD

**Discussion by Srolfe657** [josephdadams/TallyArbiter#412] [Q&A]
Hi, I'm setting up TallyArbiter to work with OBS on the same Windows PC and no matter what I try I can't get it to see my scenes or sources. I have the websocket plugin installed and TallyArbiter sees it connect as a source. When I add a device and edit the device source the only options I can get to come up are {{Streaming}} and {{Recording}}. I've tried both 127.0.0.1 and the PCs local IP. I've disabled windows firewall, I tried restarting both TallyArbiter and OBS as well as the computer itself.

Do I need to put some sort of special brackets around the Address in the source? I'm using a m5stickc and it works perfectly for streaming/recording options and for the Test Mode as well but I just can't get it to change when I select a different scene in OBS.

Any help would be great! Thank you!

**Comment by MatteoGheza** [comment]
What version of OBS and TallyArbiter are you using?

**Reply by Srolfe657** [reply]
OBS = 27.2.3 and TallyArbiter= 2.0.16

**Reply by hrueger** [reply]
And what version of the websocket plugin? v4 or v5?

**Reply by Srolfe657** [reply]
4.9.1 for the websocket

**Reply by hrueger** [reply]
Hm, that's strage, because those versions should work.
If you want, you can try the latest beta of TallyArbiter (Version 3) because it contains a lot of improvements and bug fixes. Please notice that you will need a new listener client version for that.
